homeassistant/helpers/state.py

```diff
@@ -53,42 +32,20 @@
 GROUP_DOMAIN = 'group'
 
 # Update this dict of lists when new services are added to HA.
-# Each item is a service with a list of required attributes.
-SERVICE_ATTRIBUTES = {
-    SERVICE_PLAY_MEDIA: [ATTR_MEDIA_CONTENT_TYPE, ATTR_MEDIA_CONTENT_ID],
-    SERVICE_MEDIA_SEEK: [ATTR_MEDIA_SEEK_POSITION],
-    SERVICE_VOLUME_MUTE: [ATTR_MEDIA_VOLUME_MUTED],
-    SERVICE_VOLUME_SET: [ATTR_MEDIA_VOLUME_LEVEL],
-    SERVICE_NOTIFY: [ATTR_MESSAGE],
-    SERVICE_SET_AWAY_MODE: [ATTR_AWAY_MODE],
-    SERVICE_SET_FAN_MODE: [ATTR_FAN_MODE],
-    SERVICE_SET_FAN_MIN_ON_TIME: [ATTR_FAN_MIN_ON_TIME],
-    SERVICE_RESUME_PROGRAM: [ATTR_RESUME_ALL],
-    SERVICE_SET_TEMPERATURE: [ATTR_TEMPERATURE],
-    SERVICE_SET_HUMIDITY: [ATTR_HUMIDITY],
-    SERVICE_SET_SWING_MODE: [ATTR_SWING_MODE],
-    SERVICE_SET_HOLD_MODE: [ATTR_HOLD_MODE],
-    SERVICE_SET_OPERATION_MODE: [ATTR_OPERATION_MODE],
-    SERVICE_SET_AUX_HEAT: [ATTR_AUX_HEAT],
-    SERVICE_SELECT_SOURCE: [ATTR_INPUT_SOURCE],
-    SERVICE_SEND_IR_CODE: [ATTR_IR_CODE],
-    SERVICE_SELECT_OPTION: [ATTR_OPTION],
-    SERVICE_SET_COVER_POSITION: [ATTR_POSITION],
-    SERVICE_SET_COVER_TILT_POSITION: [ATTR_TILT_POSITION]
+# Each item is a service which has unique named attribute for
+# the domain.
+SERVICE_WHITELIST = {
```

Comment:
Do we still need this?

Reply:
Nope. Just so I remember what components remain to do.